package.json

```diff
@@ -21,7 +21,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.22",
     "@fortawesome/free-brands-svg-icons": "^5.10.2",
     "@fortawesome/react-fontawesome": "^0.1.4",
-    "@smartcitiesdata/react-discovery-ui": "^2.1.8",
+    "@smartcitiesdata/react-discovery-ui": "2.1.9",
```

Comment:
I think removing the caret here to make it explicit is fine

Reply:
Considering the extent of changes we usually make, and that we have to manually deploy anyway, figured it was the right move for now. Would really like to see react_discovery_ui just pulled into this project at some point